.github/workflows/doc-build-test.yml

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: DOC CI
```

Comment:
Can we include the link check ci in the check md file along with.

Reply:
Good idea. When executing `npm run build`, it will verify whether the file referenced in sidebar.json exists. Indeed, the link in the .md file has not yet been verified. Let me try to see if it is easy to implement.

Reply:
@yuluo-yx  I'm really sorry, I made a mistake before. According to the configuration in `docusaurus.config.js`, if a dead link occurs, the build process will be terminated, and the purpose of link verification can be achieved.

```javascript
onBrokenLinks: 'throw',
onBrokenMarkdownLinks: 'throw',
```

Reply:
got it.